playbooks/allocate_pubcloud.yml

```diff
@@ -39,6 +39,19 @@
           {%- set region_list = (fallback_regions | replace(',', ' ')).split() | map('trim') | map('upper') | select | shuffle %}
           {{- (region_list | length == 1) | ternary(region_list, region_list | difference([regions_shuff[0]])) }}
 
+    - name: Determine the latest image name when required
+      when:
+        - image | match('^nodepool-.*')
+      block:
+        - name: Get the list of images
+            os_image_facts:
+            cloud: "{{ cloud_name }}"
+            region_name: "{{ regions_shuff[0] }}"
```

Comment:
We're seeing a syntax error on `os_image_facts:`, need to indent `cloud` and `region_name` properties?

Reply:
Yep, ``os_image_facts`` was incorrectly indented.